.eslintrc.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:prettier/recommended',
+    'plugin:import/errors',
+    'plugin:import/warnings',
+    'prettier/@typescript-eslint',
+  ],
+  env: {
+    browser: true,
+    mocha: true,
+  },
+  plugins: ['import', '@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/prefer-interface': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-unused-vars': 'error',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
explicit any is necessary now.
For example, custom `window` member is invalid in types. It need original declaration to fix it. (like as below)
```
const w = window as any
w.q = function......
```